content/en/network_performance_monitoring/network_page.md

```diff
@@ -117,9 +117,9 @@ You can configure the columns in your table using the `Customize` button at the
 
 Congifure the traffic shown with the `Filter Traffic` button.
 
-{{< img src="network_performance_monitoring/network_page/filter_traffic_toggles.png" alt="Flow Details"  style="width:80%;">}}
+{{< img src="network_performance_monitoring/network_page/filter_traffic_toggles_v2.png" alt="Flow Details"  style="width:80%;">}}
 
-Datadog Agent traffic is shown by default. To narrow down your view to non-Datadog traffic _only_, toggle off `Show Datadog Traffic`.
+External traffic (to public IPs) and Datadog Agent traffic is shown by default. To denoise your view, you can choose to toggle off the `Show Datadog Traffic` and `Show External Traffic` toggles.
```

Comment:
```suggestion
External traffic (to public IPs) and Datadog Agent traffic is shown by default. To simplify your view, you can choose to toggle off the `Show Datadog Traffic` and `Show External Traffic` toggles.
```

Reply:
What do you think about 'narrow down'? The goal isn't to make the view simpler, just dense with more relevant information.

Reply:
sure, updated my suggestion